dns_editor.php

```diff
@@ -108,11 +108,22 @@
 $type_select_values = array(
 	'A',
 	'AAAA',
-	'NS',
+	'CAA',
+	'CNAME',
+	'DNAME',
+	'DNSKEY',
+	'DS',
+	'LOC',
 	'MX',
+	'NS',
+	'NSEC',
+	'NSEC3',
+	'PTR',
+	'RP',
+	'RRSIG',
 	'SRV',
+	'SSHFP',
```

Comment:
please do not add various types for which there is no validation...could break dns-feature

Reply:
haha, okay. I did not even know about the validations. But I'd rather add validations and finally have these RR types, than remove them again...

Reply:
You're very welcome to do so :)

Reply:
Haha yeah, already on it.

Reply:
But i would not put too much effort in very special rarely used entries....as dns is not very much used (own nameserver is not for everyone, not many people do that)

Reply:
> as dns is not very much used (own nameserver is not for everyone, not many people do that)

I've seen in the forum, that this is your opinion, but I for example **am** running my DNS cluster and need some of the records, though not all of them, that's right.

Reply:
I wouldn't call it opinion - rather experience and knowledge of froxlor users, quite some years now including support etc. - just had ONE who was actually using DNS...just saying :)

Reply:
don't get me wrong here - your work is very appreciated - just wanted to tell you that putting too much effort in a very seldomly used feature can be frustrating ;) Maybe for now add only the entries you are using or which seem to be necessary for a "normal" operation

Reply:
> I wouldn't call it opinion - rather experience and knowledge of froxlor users, quite some years now including support etc. - just had ONE who was actually using DNS...just saying :)

Okay, fair enough. Still, I'm running my own DNS servers and some of the added types are indeed very useful, while others like the the PTR were added by me without thinking much, I'll remove that one again, since I'm not running a global in.addr.arpa domain, only locally and I'm not using froxlor for local networks anyway, so...

Reply:
Should be fixed in 358ca61a2698a137aa64b48dba24d254a016b229, right?